### DISCUSSION TITLE
[Recommendation Issue] Copilot recommend me Korean profanity.

### DISCUSSION THREAD

**Discussion by ChanHHOO** [community/community#128753] [Copilot Conversations]
### Select Topic Area

Product Feedback

### Body

Hi guys,
When I write code using Copilot, I got profanity like image.
That Korean sentence mean "F***, what is this?". 
I hope to implement a filtering feature for Korean profanity. Seeing swear words during development has caused my concentration to drop suddenly.

<img width="502" alt="image" src="https://github.com/community/community/assets/39542964/4b18bdb0-c246-451b-bacb-a26501249cd5">

**Comment by ayim** [comment]
Hi @ChanHHOO, this looks like VSCode to me, but can you confirm the IDE that you're using?

**Reply by ChanHHOO** [reply]
Hi ayim, you are right. My IDE is VSCode 🫡

**Reply by ayim** [reply]
In that case, @Akash1134 will take a look and help the feedback reach the appropriate team

**Reply by Akash1134** [reply]
Hi @ChanHHOO Thank you for your report. We've shared this with the appropriate team internally and they are investigating this.

**Reply by Akash1134** [reply]
Hi @ChanHHOO We'd like to know your workflow in Copilot and the task you were trying to achieve so that we can look into the issue in detail. Thanks!

**Reply by ChanHHOO** [reply]
I was writing comments for my interface in TypeScript code.
My interface contains user information.
I wrote comments at the top of the interface code.

When I typed ‘ㅅ’, I got “ㅅㅂ이거 뭐냐“.
‘ㅅㅂ’ means ‘Fuck’

If you need any more information, please let me know.

**Reply by Akash1134** [reply]
Thank you! We have shared the details with the concerned team.

**Comment by ChrisWiles** [comment]
@ChanHHOO Its a LLM, not a big deal. Ignore it

**Reply by ChanHHOO** [reply]
Yeah I agree too. However I wish that AI service do not recommend profanity 🤤
